Services/CO.CDP.Person.WebApi/Api/Person.cs

```diff
@@ -78,6 +99,7 @@ await useCase.Execute(personId)
             return Results.NoContent();
         })
             .Produces(204)
+            .Produces(404)
```

Comment:
Should 404 be returned from Delete?
https://stackoverflow.com/questions/6439416/status-code-when-deleting-a-resource-using-http-delete-for-the-second-time

Reply:
I guess we will handle the UI and will not let the users delete the same resource twice. I will remove it now.